Street Trees/STOrderFormNavigationViewController.swift

```diff
@@ -64,6 +66,15 @@ STAddressFormViewControllerDelegate {
         if let treeViewController = viewController as? STSelectTreeViewController {
             treeViewController.delegate = self
         }
+        
+        if let confirmationViewController = viewController as? STConfirmationPageViewController {
```

Comment:
only assign the delegate in this if let. By conforming the confirmationViewController to the TreeDescription, Address, and Contact protocols it will be given the correct objects when it is placed on the navigation stack.